package.json

```diff
@@ -40,6 +40,7 @@
     ]
   },
   "devDependencies": {
+    "dotenv-webpack": "^7.0.1",
```

Comment:
why not use the latest version? `8.2.0`

Reply:
I think this is the latest version 
![image](https://user-images.githubusercontent.com/35818757/110243536-1706b880-7f5b-11eb-8a1e-e4d201cfc1a0.png)

Reply:
why `dotenv-webpack` instead of `dotenv`?
https://www.npmjs.com/package/dotenv

Reply:
`dotenv` is used if you want to use variables from your .env file in the webpack.config.js directly and it behaves just as any other javascript file while being run. `dotenv-webpack` plugin instead inserts the variables into your bundled code, which is enough for the deploy

Reply:
If you looked into the file you are changing `fetch.js` it is doing it for once, once it fetches the details those env vars are used at all. If it runs before starting the react-scripts build. Essentially, just because there was a vulnerability in inserting the env vars in build code I hade to create #333 so it should be fine to use `dotenv` here.

Reply:
ok, changed it :)